tests/test_map_plugin.py

```
@@ -1013,6 +1036,31 @@ async def run_test():
 
         asyncio.run(run_test())
 
+    @patch("mmrelay.matrix_utils.send_image")
+    @patch("mmrelay.plugins.map_plugin.get_map")
+    @patch("mmrelay.plugins.map_plugin._connect_meshtastic_async")
+    @patch("mmrelay.matrix_utils.connect_matrix")
+    def test_handle_room_message_extract_args_returns_none(
+        self, mock_connect_matrix, mock_connect_mesh, mock_get_map, mock_send_image
```

Ruff (0.15.10)

[warning] Unused method argument: `mock_connect_matrix`

(ARG002)